lldb/include/lldb/lldb-enumerations.h

```diff
@@ -1347,6 +1347,12 @@ enum DebuggerBroadcastBit {
   eBroadcastBitProgressCategory = (1 << 3),
 };
 
+enum Severity {
```

Comment:
Would `enum class` prevent people from accidentally casting this to an unrelated enum with similar purpose but potentially different values?

Reply:
It would, but `lldb-enumerations.h` exclusively uses `enum` and I think consistency outweighs the small benefit of using `enum class`. FWIW this came up previously in https://github.com/llvm/llvm-project/pull/80167#issuecomment-2043721298 too.